wiki/reinstalling-microsoft-store.md

```diff
@@ -0,0 +1,61 @@
+# Reinstalling Microsoft Store
+
+Microsoft Store is used to download applications. But it may sometimes glitch out and disappear. This article will cover the steps of reinstalling it.
+
+## Method 1: Resetting the Microsoft Store Cache
+
+There are two methods to clear out the Microsoft Store Cache. 
+
+### Method 1: Resetting Microsoft Store through the Settings App
+
+1. Open the Settings app. You can do this by searching for it in the Start menu, pressing <kbd>Win</kbd> + <kbd>I</kbd>, or pressing the Settings icon above the power button.
+
+    ![](img/reinstalling-microsoft-store/opensettings.png)
+
+2. Select "Apps".
+
+    <img src="img/reinstalling-microsoft-store/apps.png" style="width:300px;">
+
+3. Find "Microsoft Store" and click advanced options.
+
+    <img src="img/reinstalling-microsoft-store/microsoftstore.png" style="width:500px">
+
+> If you can't find Microsoft Store here, you wll have to reinstall Microsoft Store completely. Follow Method 2 to reinstall Microsoft Store
+
+4. Scroll down and click the reset button. Then wait for a check mark to appear.
+
+    <img src="img/reinstalling-microsoft-store/resetmicrosoftstore.png" style="width:500px">
+
+### Method 2: Resetting Microsoft Store through WSReset
```

Comment:
WSReset has very dubious success rates and as far as I'm aware doesn't even function correctly anymore. Not saying this should be removed but it should probably be noted somewhere that you shouldn't expect much